backend/overstroomik_service/geoserver.py

```diff
@@ -0,0 +1,152 @@
+"""
+This class uses the geoserver-api and geoserver layers to find
+location information by rd coordinates
+"""
+
+from typing import Optional
+import httpx
+
+from overstroomik_service.auto_models import Data
+from overstroomik_service.config import settings
+
+ERROR_GENERAL_NOER = "no-error"
```

Comment:
We now have multiple status codes spread over the code. I think one place (errors.py?) and maybe making it into an Enum would make things more readable/documentable.

Reply:
All status codes placed in an enum class (errors.py)